notebooks/community/managed_notebooks/Recommender Systems/Recommender-system_notebook.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        .config("spark.jars", "gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar")
</pre><p>Who owns this bucket?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>